default.nix

```diff
@@ -8,6 +8,33 @@ in
     dart = callPackage ./pkgs/dart {};
     flutter = callPackage ./pkgs/flutter {};
     flutter-patch = callPackage ./pkgs/flutter-patch {};
+    all-tools = cli-tools ++ clojure-tools ++ desktop-tools;
+    cli-tools = with pkgs; [
+      awscli
+      fzf
+      gettext
+      jq
+      kubectl
+      nodejs-10_x
+      nss
+      nssTools
+      openfortivpn
+      openssl
+      python37Full
+      # TODO: ruby is installed by Ansible, but I never saw it used in Nubank
+      # ruby
+    ];
+    clojure-tools = with pkgs; [
+      apacheKafka
```

Comment:
Why do we need to build kafka? do we run kafka local to check some topic or something like this?

Reply:
We just need the `kafka` tools, not kafka itself.

Reply:
Actually, we only use in very specific situations, so I removed it.